x-pack/solutions/observability/plugins/infra/public/pages/metrics/inventory_view/components/layout.tsx

```diff
@@ -60,11 +64,26 @@ export const Layout = React.memo(({ interval, nodes, loading }: Props) => {
   } = useWaffleOptionsContext();
   const { currentTime, jumpToTime, isAutoReloading } = useWaffleTimeContext();
   const { applyFilterQuery } = useWaffleFiltersContext();
+  const { data: timeRangeMetadata } = useTimeRangeMetadataContext();
   const legendPalette = legend?.palette ?? DEFAULT_LEGEND.palette;
   const legendSteps = legend?.steps ?? DEFAULT_LEGEND.steps;
   const legendReverseColors = legend?.reverseColors ?? DEFAULT_LEGEND.reverseColors;
-
   const AUTO_REFRESH_INTERVAL = 5 * 1000;
+  const schemas: DataSchemaFormat[] = useMemo(
+    () => timeRangeMetadata?.schemas || [],
+    [timeRangeMetadata?.schemas]
+  );
+
+  const hasElasticSchema = schemas.includes('ecs');
+  const hasOtelSchema = schemas.includes('semconv');
+  const { isInstalled: hasEcsK8sIntegration } = useInstalledIntegration('kubernetes');
+  const { isInstalled: hasOtelK8sIntegration } = useInstalledIntegration('kubernetes_otel');
+
+  const [ecsK8sCardDismissed, setEcsK8sCardDismissed] = useState(false);
```

Comment:
This won't work as expected:
The banner/card will show again every time the user:
- Refreshes the page
- Navigates away and comes back
- Closes and reopens the browser
We should use `useLocalStorage()`

```suggestion
  const [ecsK8sCardDismissed, setEcsK8sCardDismissed] = useLocalStorage(
  'kibana.infra.ecsK8sCardDismissed',
  false
);
```